src/firefly/test/python/demo-notebook.ipynb

```diff
@@ -0,0 +1,423 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from FireflyClient import *\n",
```

Comment:
Please use `from FireflyClient import FireflyClient`. I also wonder if the module should be named `fireflyClient.py` -- I couldn't tell for sure from reading the DM Python Style guide.